npm/webpack-dev-server/cypress/e2e/angular.cy.ts

```diff
@@ -52,7 +51,26 @@ for (const project of WEBPACK_REACT) {
         )
       })
 
-      cy.get('.passed > .num').should('contain', 1)
+      cy.waitForSpecToFinish({ passCount: 1 })
+    })
+
+    it('should show compilation errors on src changes', () => {
+      cy.visitApp()
+
+      cy.contains('app.component.cy.ts').click()
+      cy.waitForSpecToFinish({ passCount: 1 })
+
+      // Create compilation error
+      cy.withCtx(async (ctx) => {
+        await ctx.actions.file.writeFileInProject(
+          ctx.path.join('src', 'app', 'app.component.ts'),
+          (await ctx.file.readFileInProject(ctx.path.join('src', 'app', 'app.component.ts'))).replace('class', 'classaaaaa'),
+        )
+      })
```

Comment:
```suggestion
      cy.withCtx(async (ctx) => {
        const componentFilePath = ctx.path.join('src', 'app', 'app.component.ts')
        await ctx.actions.file.writeFileInProject(
          componentFilePath,
          (await ctx.file.readFileInProject(componentFilePath)).replace('class', 'classaaaaa'),
        )
      })
```
Just a nit-pick about not duplicating the creation of the file path